### PR TITLE
fix pprof redirect path

### PR DESCRIPTION
**Pull request by bestgopher**
I think the redirected path can be optimized.
For example, if the path is `/debug/pprof/block/`, it better to redirect to `/debug/pprof/block` instead of `/debug/pprof`.

### CODE REVIEW

File: middleware/pprof/pprof.go [hunk 1]
```diff
@@ -58,7 +58,13 @@ func New() fiber.Handler {
 			pprofThreadcreate(c.Context())
 		default:
 			// pprof index only works with trailing slash
-			return c.Redirect("/debug/pprof/", 302)
+			if strings.HasSuffix(path, "/") {
+				path = strings.TrimSuffix(path, "/")
+			} else {
+				path = "/debug/pprof"
+			}
```
[the remark this answers is not in the document]
Comment (bestgopher) [reply middleware/pprof/pprof.go:65]: If there is no suffix, path will be `/debug/pprof`.

File: utils/bytes.go [hunk 2]
```diff
@@ -55,7 +55,7 @@ func TrimBytes(b []byte, cutset byte) []byte {
 	return b[i : j+1]
 }
 
-// EqualFold the equivalent of bytes.EqualFold
+// EqualFoldBytes the equivalent of bytes.EqualFold
```
Comment (kiyonlin) [inline utils/bytes.go:58]: Please split all irrelevant changes in another PR

**Review by kiyonlin — APPROVED** [review]

**Review by hi019 — APPROVED** [review]
